mod/qbassign/externallib.php

```diff
@@ -3573,27 +3572,28 @@ public static function create_assignment_service($courseid,$siteid,$chapterid,$t
                      $onlinetext_default = $DB->insert_record('qbassign_plugin_config', $updatesactivityonline);
                  }           
              }
-             purge_all_caches();
              $DB->set_field('qbassign', 'uid', $uid, array('id' => $returnid));
              $lti_updated = [                        
```

Comment:
Can you please change the variable name. We didn't use LTI here. So we don't want to use the variable name as 'lti_updated'.  Refer this URL [](https://www.imsglobal.org/activity/learning-tools-interoperability)